systemd/systemd.go

```diff
@@ -105,7 +105,7 @@ type RestartCondition string
 
 // These are the supported restart conditions
 const (
-	RestartNever      RestartCondition = "never"
+	RestartNever      RestartCondition = "no"
```

Comment:
It looks like the logic in this file would imply going back and forth onto a yaml file would create a broken setting.

Instead of changing it here, perhaps it would be slightly more correct to change the generation of the template?  That would mean everything in memory has it the same consistent way, and only when we output the value onto the systemd file (which is never read back) we move it into "no".